css/types/counters.json

```
@@ -1,16 +1,16 @@
 {
   "css": {
     "types": {
-      "attr": {
+      "counters": {
         "__compat": {
           "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/counters",
           "description": "<code>counters()</code>",
           "support": {
             "chrome": {
-              "version_added": true
+              "version_added": "4"
```

Again, following the WebKit change:

```suggestion
              "version_added": "1"
```